engine/graphics/src/dx12/graphics_dx12.cpp

```diff
@@ -177,17 +173,6 @@ namespace dmGraphics
         }
     }
 
-    static void CreateRootSignature(DX12Context* context, CD3DX12_ROOT_SIGNATURE_DESC* desc, DX12ShaderProgram* program)

```

Comment:
We will have to re-add this function later on to support building shaders on non-windows platforms.